src/components/Buttons/BasicSigns.jsx

```diff
@@ -1,29 +1,18 @@
 import React from 'react';
```

Comment:
- [ ] Let's please use the README file from the Microverse template. Although the one created by CRA is not bad, it's very generic and be seen as unprofessional. Let's remember that keeping our repositories documented professionally it's important as a recruiter might visit these at any given moment. Read more about how to keep a [repository professional here.](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)